scripts/install.sh

```diff
@@ -258,4 +264,22 @@ wait_for_pods_ready "-l control-plane=modelmesh-controller"
 info "Installing ModelMesh Serving built-in runtimes"
 kustomize build runtimes --load-restrictor LoadRestrictionsNone | kubectl apply -f -
 
+if [[ ! -z $user_ns_array ]]; then
+  kustomize build runtimes --load-restrictor LoadRestrictionsNone > runtimes.yaml
+  cp dependencies/user_ns.yaml .
+  sed -i "s/controller_namespace/${namespace}/g" user_ns.yaml
+
+  for user_ns in "${user_ns_array[@]}"; do
+    if ! kubectl get namespaces $user_ns >/dev/null; then
+      echo "Kube namespace does not exist: $user_ns. Will skip."
+    else 
+      kubectl label namespace ${user_ns} modelmesh-enabled="true"
+      kubectl apply -f user_ns.yaml -n ${user_ns}
```

Comment:
I think we only want to do do this in the `--quickstart` case. Also maybe we should rename it to something like `minio-storage-secret.yaml`?

Reply:
Sure. Updated.